nodes/vr_sros/vr-sros.go

```diff
@@ -171,12 +172,30 @@ func isPartialConfigFile(c string) bool {
 	return strings.Contains(strings.ToUpper(c), ".PARTIAL")
 }
 
+// isHealthy checks if the "/health" file created by vrnetlab exists and contains "0 running"
+func (s *vrSROS) isHealthy(ctx context.Context) bool {
+	ex := exec.NewExecCmdFromSlice([]string{"bash", "-c", "cat /health | perl -lne '/0 running/ && $MATCH++; END{exit 1 unless $MATCH>0}'"})
+
```

Comment:
maybe let's keep perl in the grave and use `grep '0 runnning' /health`?

Reply:
done